packages/firestore/test/unit/remote/serializer.test.ts

```diff
@@ -82,24 +75,26 @@ import {
   version,
   wrap,
   wrapObject
-} from '../../../util/helpers';
-import { ByteString } from '../../../../src/util/byte_string';
+} from '../../util/helpers';
+import { ByteString } from '../../../src/util/byte_string';
+import { isNode } from '../../util/test_platform';
+
+let verifyProtobufJsRoundTrip: (jsonValue: api.Value) => void = () => {};
+
+if (isNode()) {
```

Comment:
It seems like you could just inline the definition of `verifyProtobufJsRoundTrip` here. It doesn't seem like it would be useful in any other context because it hard codes the use of `ValueMessage`.

Reply:
I put in a separate file so we could import ProtobufJs and its types using "import" instead of "require". This gives us type safety. If I included it here, I would have to rely on the `d.t.s` files for these dependencies to use the types safely, without importing the actual sources (an import of `ProtobufJs` here would break the build for the browser tests)..